packages/itwin/one-click-lca-widget/src/components/ExportModal.tsx

```diff
@@ -94,7 +94,7 @@ const ExportModal = (props: ExportProps) => {
             if (
               currentJobStatus.job?.status === JobStatus.StatusEnum.Succeeded
             ) {
-              setJobLink(currentJobStatus?.job._links?.oneclicklca);
+              setJobLink(undefined !== currentJobStatus?.job._links?.oneclicklca?.href ? {href: currentJobStatus.job._links.oneclicklca.href} : undefined);
```

Comment:
Check if `currentJobStatus?.job._links?.oneclicklca?.href` evaluates to a falsy value instead.

Reply:
changed the condition to ```!currentJobStatus?.job._links?.oneclicklca?.href```. Because of this some non-null assertions were added.

Reply:
Yeah, my bad. It should be fine now.